### DISCUSSION TITLE
Reason behind choosing cva other other libraries like tw-classed and tailwind-variants

### DISCUSSION THREAD

**Discussion by PrajTS** [roninoss/nativewindui#4] [Q&A]
Hey @danstepanov @mrzachnugent , 

This is much suited for a discussions tab, rather than an issue. Couldnt see any discussion tab, so asking here.

**Is there reason cva was chosen over other suggested libraries like tw-classes and tailwind-variants?**

**Comment by mrzachnugent** [comment]
It's the shadcn-ui way. It's a reliable library and we do not have the need for the features offered by the other libraries. No other reason.

The other libraries seem perfectly fine to use but it does not seem worth it to change for our specific needs.
